Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/dungeon_items_weapons.yml

```diff
@@ -59,6 +59,8 @@
     #- WeaponCaseLongShotgunDoubleBarreledExpedition
     - WeaponCaseShortSawnExpedition
     - WeaponCaseLongMosinExpedition
+    - WeaponSniperCeremonial
+    - WeaponRevolverLucky
```

Comment:
This is not how entities added to this list, needs to be brought in line with other items: C2 variant ("unregistered") in a weapon case.